ldi-orchestrator/ldio-connectors/ldio-console-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdiConsoleOut.java

```diff
@@ -20,6 +20,6 @@ public LdiConsoleOut(LdiRdfWriterProperties properties) {
 
 	@Override
 	public void accept(Model model) {
-		log.info(getRdfWriter(properties).write(model));
+		log.debug(getRdfWriter(properties).write(model));
```

Comment:
I would keep this "info" since it is actually a component that you have to configure. If this is debug, then you have to run everything on debug and you need more config to actually see what you want. I also assume that this component is mainly used in development, so I would keep this as developer friendly as possible.

Reply:
Yeah yeah, accidently changes this for my test :)